frontend/src/main/scala/bloop/engine/Dag.scala

```diff
@@ -35,7 +41,19 @@ object Dag {
             project match {
               case leaf if project.dependencies.isEmpty => Leaf(leaf)
               case parent =>
-                val childrenNodes = project.dependencies.iterator.map(name => projectsMap(name))
```

Comment:
Is this where the previous `NoSuchElementException` happened?

Reply:
Yes, in the `apply` of `Map`